app/views/works/_work_module.html.erb

```diff
@@ -99,7 +99,7 @@
 
   <% if work.count_visible_comments > 0 %>
     <dt class="comments"><%= ts('Comments') %>:</dt>
-    <dd class="comments"><%= link_to work.count_visible_comments, work.chapter_total_display.to_i > 1 ? work_path(work, anchor: "comments", show_comments: true, view_full_work: "true") : work_path(work, anchor: "comments", show_comments: true) %></dd>
+    <dd class="comments"><%= link_to number_with_delimiter(work.count_visible_comments), work.chapter_total_display.to_i > 1 ? work_path(work, anchor: "comments", show_comments: true, view_full_work: "true") : work_path(work, anchor: "comments", show_comments: true) %></dd>
```

Comment:
Accidental extra change?

Reply:
i think i did it on purpose (although it was a while ago so i can't remember) because it looked weird without the delimiter and the actual non-download page has delimiters. can remove if unwanted though!

Reply:
As far as I understand, the cache key in line 94 would have to be changed for this to work properly. So I think it would be better to remove the change. Additionally, there is AO3-5777 for adding the delimiters in this file.